src/styles/todo.scss

```diff
@@ -65,21 +65,21 @@
       color: #af5b5e;
     }
   }
-  
+
   &:hover &__remove {
     opacity: 1;
   }
 
   &__title-field {
```

Comment:
Consider using a nested class for the __title-field element, like '.todo-item__title-field'